doc/devel/coding_guide.rst

```diff
@@ -41,47 +41,59 @@ Documentation
 PR Review guidelines
 ====================
 
+* Be patient and `kind <https://youtu.be/tzFWz5fiVKU?t=49m30s>`__ with
+  contributors.
+
 * If you have commit rights, then you are trusted to use them.  Please
   help review and merge PRs!
 
-* For code changes (anything in ``src`` or ``lib``) two developers
-  (those with commit rights) should review all pull requests.  If you
-  are the first to review a PR and approve of the changes use the
-  github `'approve review'
+* Documentation and examples may be merged by the first reviewer.  Use
+  the threshold "is this better than it was?" as the review criteria.
+
+* For code changes (anything in ``src`` or ``lib``) at least two
+  developers (those with commit rights) should review all pull
+  requests.  If you are the first to review a PR and approve of the
+  changes use the github `'approve review'
   <https://help.github.com/articles/reviewing-changes-in-pull-requests/>`__
-  tool to mark it as such.  If you are a subsequent reviewer and you
-  approve, either merge (and backport if needed) or select ``'approve
-  review'``.
+  tool to mark it as such.  If you are a subsequent reviewer please
+  approve the review and if you think no more review is needed, merge
+  the PR.
 
   Ensure that all API changes are documented in
   :file:`doc/api/api_changes` and significant new features have and
   entry in :file:`doc/user/whats_new`.
 
-* Documentation and examples may be merged by the first reviewer.  Use
-  the threshold "is this better than it was?" as the review criteria.
-
-* Make sure the Travis, Appvyor, and codecov tests are passing before
-  merging.
+* Make sure the Travis, Appvyor, circle, and codecov tests are passing
+  before merging.
 
   - Whenever a pull request is created or updated, Travis and Appveyor
     automatically runs the test suite on all versions of Python
     supported by Matplotlib.  The `tox` support in Matplotlib may be
```

Comment:
Not sure what this phrase means.  Why is `tox` support useful locally?  

Overall, while you are looking at this, it'd be useful to have a short section explaining what all the tests do (and have folks update this as the tests change). The different Codecov tests are still a bit of a mystery to me, though I think I have a basic understanding of them.  LGTM - who knows what thats testing, and if the current failures matter at all.